sample-key-extractors/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM haystack-pipes-kafka-producer:latest
+MAINTAINER Haystack <haystack@expedia.com>
+
+RUN mkdir -p ${APP_HOME}/extractors/
+COPY target/haystack-sample-key-extractors.jar ${APP_HOME}/extractors/
+RUN ls ${APP_HOME}/extractors/
```

Comment:
Is the intention to just show a list of extractors? or why is this one necessary?